sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -556,14 +713,12 @@ public async Task StartRecognizeCustomFormsWithoutLabels(bool useStream)
         }
 
         [Test]
-        [TestCase(true, true)]
-        [TestCase(true, false)]
-        [TestCase(false, true, Ignore = "Service returning 'Unsupported media type' error.")]
-        [TestCase(false, false, Ignore = "Service returning 'Unsupported media type' error.")]
-        public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForms(bool useStream, bool includeTextContent)
+        [TestCase(true)]
+        [TestCase(false, Ignore = "Service returning 'Unsupported media type' error.")]
```

Comment:
Service-side issue. Service and SDK teams are already aware of it.

Reply:
out curiosity, is this something the service team will fix? do we know the ETA?

Reply:
It will be good to track this `Ignore` test in an issue, so later we don't forget to check with the service team the status of it and enable the tests again

Reply:
> is this something the service team will fix?

Not as far as we know. There's a thread for that in the Form Recognizer service channel. According to the service team, this is a GitHub issue.
```
GitHub is not responding with a correct Content-Type
Content-Type: application/octet-stream
```

It's still a bit weird to understand what's going on since this only happens with custom recognition **without** labels. Recognition with labels works just fine.

I'll open issues for the ignored tests once I merge this PR. I'll add them to our backlog milestone.

Reply:
#12319

Reply:
> It's still a bit weird to understand what's going on since this only happens with custom recognition without labels. Recognition with labels works just fine.

Do we know why this is?  It would be worth understanding the difference between the endpoints so we can communicate this to the service team.

Reply:
> Do we know why this is?
We don't, but they're aware. Krista has mentioned it in the FR channel. I'll send you a link to the discussion.